### ISSUE TITLE
Just Build Examples

### ISSUE THREAD

**Issue by alexhadjilambris** [gcanti/fp-ts#916]
Hi I have a beginner question. I would like simply to build the scripts in the tutorials folder, but am not able to do this. Tutorial number one builds, but number two does not.

Can you please tell me whether you think that it is likely that I have too old a version of tsc, or whether my compile flags are not correct?

Here is the message:

<pre><font color="#8AE234"><b>alexi@workstation</b></font>:<font color="#729FCF"><b>~/work/fp-ts/tutorials</b></font>$ tsc fp-ts-to-the-max-II.ts 
<font color="#34E2E2">../src/Array.ts</font>:<font color="#FCE94F">148</font>:<font color="#FCE94F">14</font> - <font color="#EF2929">error</font><font color="#555753"> TS2322: </font>Type &apos;undefined[]&apos; is not assignable to type &apos;never[]&apos;.
  Type &apos;undefined&apos; is not assignable to type &apos;never&apos;.

<span style="background-color:#D3D7CF"><font color="#2E3436">148</font></span> export const empty: Array&lt;never&gt; = []
<span style="background-color:#D3D7CF"><font color="#2E3436">   </font></span> <font color="#EF2929">             ~~~~~</font>

Found 1 error.

<font color="#8AE234"><b>alexi@workstation</b></font>:<font color="#729FCF"><b>~/work/fp-ts/tutorials</b></font>$ 
</pre>

**Comment by alexhadjilambris** [comment]
I don't mean to be a pest, but I've decided to reopen this.

I checked the documentation that tsc checks the parent directory for a tsconfig.json file, so I think that compile flags must not be the problem?

This is appears to easy to fix, but I just expect to be able to pull the repository and that the tutorials would build.

Is it deliberately broken, or is it just that I don't have enough experience to see what's wrong with it?
